leaks_test.go

```diff
@@ -83,6 +84,57 @@ func TestVerifyNone(t *testing.T) {
 	bg.unblock()
 }
 
+func TestIgnoreCurrent(t *testing.T) {
+	t.Run("Should ignore current", func(t *testing.T) {
+		defer VerifyNone(t)
```

Comment:
```suggestion
		defer VerifyNone(t)

```

(please leave a new line after `defer`)

Reply:
Fixed

Reply:
I think this is still missing (new line after the `defer`)

Reply:
oops! Now is fixed, thanks :)